relauncher.py

```diff
@@ -1,11 +1,61 @@
 import os, time
 
+# USER CHANGABLE ARGUMENTS
+
+# Change to `True` if you wish to enable these common arguments
+
+# Run upscaling models on the CPU
+extra_models_cpu = False
+
+# Automatically open a new browser window or tab on first launch
+inbrowser = False
+
+# Run Stable Diffusion in Optimized Mode - Only requires 4Gb of VRAM, but is significantly slower
+optimized = False
+
+# Run in Optimized Turbo Mode - Needs more VRAM than regular optimized mode, but is faster
+optimized_turbo = False
+
+# Creates a public xxxxx.gradio.app share link to allow others to use your interface (requires properly forwarded ports to work correctly)
+share = False
+
+
+# Enter other `--arguments` you wish to use - Must be entered as a `--argument ` syntax
+other_arguments = ""

```

Comment:
*additional arguments
I'd also argue that you can put one here for example to  not have an empty string

Reply:
I'm not sure what you mean by an empty string? Just on it's own it doesn't do anything. It's only when a `--argument` is entered into that string field that it matters. If a user messes up and just does `argument` it will throw an error since it'll be a broken prompt. But, that's why I changed the comment above it to try and help users understand that.

Reply:
I did rename it to be `additional_variables` though. That does make more sense.